src/internal/operations/backup.go

```diff
@@ -264,9 +265,11 @@ func (op *BackupOperation) do(
 		return nil, clues.Wrap(err, "producing manifests and metadata")
 	}
 
-	_, lastBackupVersion, err := lastCompleteBackups(ctx, op.store, mans)
-	if err != nil {
-		return nil, clues.Wrap(err, "retrieving prior backups")
+	if canUseMetaData {
```

Comment:
The name of this check makes it feel unrelated to the internals.  A comment explaining how this relatets, eg: "if canUseMetaData is false then the conditions are ..." would be helpful.  And maybe a rename of the value.

Reply:
Keeping this as is to expedite corso fixes. @meain could you please follow up on this? :)